tests/selftest.rs

```diff
@@ -21,7 +21,7 @@ struct TestPackageMetadata {
 
 #[test]
 fn metadata() {
-    let metadata = cargo_metadata::metadata(None).unwrap();
+    let metadata = cargo_metadata::metadata(None::<&Path>).unwrap();
```

Comment:
hm... this is a breaking change, and the `None` case seems like a very common case

I'll hold off until the next breaking change

Reply:
Maybe we should just have two different functions instead of an `Option` argument? What do you think about `metadata` acting like the `None` variant and having no args and a `metadata_for_manifest` function that just takes an `AsRef<Path>`?

Reply:
I'm fine with that if that's what you want!